### ISSUE TITLE
Data saving.

### ISSUE THREAD

**Issue by Fertanima** [zt64/Hyperion#30]
Save data by blocking thumbnails and other images.

**Comment by daBroma** [comment]
Yeah pretty sure a data saver option is in the works but this app is far from being complete.
